### ISSUE TITLE
css integration breaking code

### ISSUE THREAD

**Issue by cconnerj** [ga-wdi-exercises/project3#480]
https://github.com/zonekid/Delmarvalous/tree/updatefunction

- The only thing that has changed from the functioning comment create, was adding the style.css.

-error when trying to click on the comment text area in the species show page i get 
        -Could not resolve 'specieIndex' from state ''
    at Object.y.transitionTo (angular-ui-router.min.js:7)

- We do not have a specieIndex state so i dont know exactly what is going on

- I believe its because we are calling something that isnt allowed in our controller, but everytime i comment stuff out that isnt in the css that works with the comment create, it still runs that error

- So basically something in the style.css is creating that error it seems like 

-if you copy and paste the below css code in the style.css the comments will create properly 

-------UPDATE: got rid of the error but cannot even click into the textarea and create

**Comment by cconnerj** [comment]
.specieshowcat {
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  border: solid orange 1px;

}
.specieshowcat * {
  color: red;
  border: 1px solid black;
  text-align: center;
}
.flexin {
  display:flex;
  justify-content: center;

}

html {
  position: relative;
  min-height: 100%;
}
.test * {
   color: orange;
   font-family: "Special Elite";
}

.welcome {
    font-family: 'Special Elite', cursive;
    color: red;
}


body {
    background-image: url("turtle.jpg");
    background-size: cover;
    margin: 0 0 100px;
}

.map-wrapper {
  display: flex;
  justify-content: center;
}

.map-wrapper a {
  color: white;
}

.map {
  background-image: url("map-of-states.jpg");
  background-repeat: no-repeat;
/*  background-size: contain;*/
  height: 900px;
  width: 1200px;
}

.specie-listing-wrapper {
  display: flex;
  height: 20px;
  position: relative;
  width: 200px;
}

.specie-icon {
  background-image: url("green-marker.png");
  background-size: contain;
  /*border: solid black 1px;*/
  display: flex;
  height: 40px;
  justify-content: center;
  width: 40px;
}

.click-target {
  /*background-color: gray;*/
  height: 40px;
  width: 20px;
}

.specie-listing {
  align-items: center;
  background-color: #53bba9;
  border: solid black 2px;
  border-radius: 20%;
  display:flex;
  height: 40px;
  justify-content: center;
  width: 120px;
}

.specie-listing-wrapper:nth-of-type(1) {
  /*background-color: red;*/
  left: 835px;
  top: 410px;
}

.specie-listing-wrapper:nth-of-type(2) {
/*  background-color: orange;*/
  left: 900px;
  top: 320px;
}

.specie-listing-wrapper:nth-of-type(3) {
  /*background-color: yellow;*/
  left: 950px;
  top: 280px;
}

.specie-listing-wrapper:nth-of-type(4) {
  /*background-color: green;*/
  left: 860px;
  top: 300px;
}

.specie-listing-wrapper:nth-of-type(5) {
  /*background-color: blue;*/
  left: 905px;
  top: 310px;
}

.specie-listing-wrapper:nth-of-type(6) {
  /*background-color: purple;*/
  left: 840px;
  top: 330px;
}

.specie-listing-wrapper:nth-of-type(7) {
  /*background-color: brown;*/
  left: 740px;
  top: 340px;
}

.specie-listing-wrapper:nth-of-type(8) {
  /*background-color: gray;*/
  left: 900px;
  top: 330px;
}

.specie-listing-wrapper:nth-of-type(9) {
  /*background-color: pink;*/
  left: 680px;
  top: 290px;
}

.specie-listing-wrapper:nth-of-type(10) {
  /*background-color: black;*/
  left: 600px;
  top: 250px;
}

.specie-listing-wrapper:nth-of-type(11) {
  /*background-color: red;*/
  left: 620px;
  top: 200px;
}

.specie-listing-wrapper:nth-of-type(12) {
  /*background-color: orange;*/
  left: 650px;
  top: 200px;
}

.specie-listing-wrapper:nth-of-type(13) {
  /*background-color: yellow;*/
  left: 720px;
  top: 140px;
}

.specie-listing-wrapper:nth-of-type(14) {
  /*background-color: green;*/
  left: 705px;
  top: 220px;
}

.specie-listing-wrapper:nth-of-type(15) {
  /*background-color: blue;*/
  left: 685px;
  top: 220px;
}

.specie-listing-wrapper:nth-of-type(16) {
  /*background-color: purple;*/
  left: 540px;
  top: 170px;
}

.specie-listing-wrapper:nth-of-type(17) {
  /*background-color: brown;*/
  left: 490px;
  top: 130px;
}

.specie-listing-wrapper:nth-of-type(18) {
  /*background-color: gray;*/
  left: 490px;
  top: 150px;
}

.specie-listing-wrapper:nth-of-type(19) {
  /*background-color: pink;*/
  left: 495px;
  top: 150px;
}

.specie-listing-wrapper:nth-of-type(20) {
  /*background-color: black;*/
  left: 760px;
  top: 140px;
}

.specie-listing-wrapper:nth-of-type(21) {
  /*background-color: red;*/
  left: 780px;
  top: 140px;
}

.specie-listing-wrapper:nth-of-type(22) {
  /*background-color: orange;*/
  left: 680px;
  top: 140px;
}

.specie-listing-wrapper:nth-of-type(23) {
  /*background-color: yellow;*/
  left: 690px;
  top: 140px;
}

.specie-listing-wrapper:nth-of-type(24) {
  /*background-color: limegreen;*/
  left: 690px;
  bottom: 160px;
}

.specie-listing-wrapper:nth-of-type(25) {
  /*background-color: blue;*/
  left: 700px;
  bottom: 150px;
}

.specie-listing-wrapper:nth-of-type(26) {
  /*background-color: purple;*/
  left: 710px;
  bottom: 150px;
}

.birds  {
    background-image: url(sky.jpg);
    min-height: 900px;
    background-size: cover;
}
img {
    max-width: 500px;
    max-height: 500px;
}

footer {
    position: absolute;
    left: 0;
    bottom: 0;
    height: 100px;
    width: 100%;
    font-size: 10px;
    text-align: center;
}

h3{
  margin-bottom: -10px;

}





/*body::after {
    content: "";
    background-image: url(collage.jpg);
    opacity: 0.5;
    top: 0;
    left: 0;
    bottom: 0;
    right: 0;
    position: absolute;
    z-index: -1;
}*/
/*main {
    min-height: 100vh;
    padding-bottom: 25px;
}*/

**Comment by jsm13** [comment]
CSS shouldn't be causing an js error. The error `Could not resolve 'specieIndex' from state...` is a router error. Looks like there previously was a `specieIndex` state. Could it be that old code with an s-ref to `specieIndex` got merged?

**Comment by cconnerj** [comment]
Yeah we found that out, now would this be the right syntax? or should we  put the include into an array
```rb
        respond_to do |format|
            format.json { render json: @specie, include: :comments, include: :category }
```

**Comment by jsm13** [comment]
putting those in an array would be better (https://www.tigraine.at/2011/11/17/rails-to_json-nested-includes-and-methods)
